app/components/blacklight/metadata_field_layout_component.rb

```diff
@@ -3,7 +3,8 @@
 module Blacklight
   class MetadataFieldLayoutComponent < ::ViewComponent::Base
     with_collection_parameter :field
-    with_content_areas :label, :value
+    renders_one :label
+    renders_one :value
```

Comment:
I'm idly wondering if `renders_many :value` would be helpful, but i'm still trying to understand the new slots approach..

regardless, do we need to bump the view_component minimum version?

Reply:
Are they used by anything? I didn't see them used in Blacklight, and they don't have any tests or documentation that I could find, so I'm having trouble visualizing how they would be used.